png-resizer/app/lib/Im4Java.scala

```diff
@@ -24,15 +25,15 @@ object Im4Java {
     baos.toByteArray
   }
 
-  def resizeBufferedImage(width: Int) = {
+  def resizeBufferedImage(width: Int)(imageBytes: Array[Byte]) = Future {
     val operation = new IMOperation
 
     operation.addImage("-")
-    operation.sharpen(1.0)
     operation.resize(width)
-    operation.quality(100)
+    operation.sharpen(1.0)
+    operation.quality(0)
```

Comment:
yeah it's not really a quality as PNGs are always lossless, it is just which kind of compression to use. It is just two independent digits between 0-9 encoded into an int (although one of them is more of an enum than a scale), so the int should be between 00 and 99. Since we are about to decompress it again using pngquant, so I set it to the quickest which is 00 by looking at the docs.